egov-persister/household-persister.yml

```diff
@@ -93,4 +93,48 @@ serviceMaps:
             - jsonPath: $.*.address.pincode
             - jsonPath: $.*.address.buildingName
             - jsonPath: $.*.address.street
-            - jsonPath: $.*.address.locality.code
\ No newline at end of file
+            - jsonPath: $.*.address.locality.code
+    - version: 1.0
+      description: Persists household member.
+      fromTopic: save-household-member-topic
+      isTransaction: true
+      queryMaps:
+        - query: INSERT INTO HOUSEHOLD_MEMBER(id, tenantId, individualId, householdId, isHeadOfHousehold, additionalDetails, createdBy, lastModifiedBy, createdTime, lastModifiedTime, rowVersion, isDeleted) VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?);
+          basePath: $.*
+          jsonMaps:
+            - jsonPath: $.*.id
+            - jsonPath: $.*.tenantId
+            - jsonPath: $.*.individualId
+            - jsonPath: $.*.householdId
+            - jsonPath: $.*.isHeadOfHousehold
+            - jsonPath: $.*.additionalFields
+              type: JSON
+              dbType: JSONB
+            - jsonPath: $.*.auditDetails.createdBy
+            - jsonPath: $.*.auditDetails.lastModifiedBy
+            - jsonPath: $.*.auditDetails.createdTime
+            - jsonPath: $.*.auditDetails.lastModifiedTime
+            - jsonPath: $.*.rowVersion
+            - jsonPath: $.*.isDeleted
+    - version: 1.0
+      description: Update household member.
+      fromTopic: update-household-member-topic
+      isTransaction: true
+      queryMaps:
+        - query: UPDATE HOUSEHOLD_MEMBER SET individualId=?, householdId=?, isHeadOfHousehold=?, additionalDetails=?, createdBy=?, lastModifiedBy=?, createdTime=?, lastModifiedTime=?, rowVersion=?, isDeleted=? WHERE individualId=? AND householdId=?;
```

Comment:
1. once a row has a set householdId and individualId, we should not allow it to be overwritten to another household/individual id.
2. createdBy, createdTime should not be changed in update
3. Should we add a where clause `and isDeleted = false` so that updates can only be made to undeleted records?
cc: @jayantp-egov @roopeshegov

Reply:
Done for all 3 points.